docs/_src/usage/usage/domain_adaptation.md

```diff
@@ -69,7 +69,73 @@ Annotate also supports two different workflows:
 
 ## User Feedback
 
-A simpler and faster process to retrain models is to utilise user feedback.
-Users can give thumbs up or thumbs down to search results through the Rest API
-and these labels will be stored in the `DocumentStore`
-where they can already be used to retrain the `Reader`.
+A simpler and faster process to retrain models is to utilise user feedback. 
+Dedicated annotation work can be costly and time consuming 
```

Comment:
Not sure if these "manual" line breaks render nicely in the documentation ...

Reply:
Checked in my md renderer and also after Gatsby rendering and it looks fine!